reporters/junit_reporter_test.go

```diff
@@ -255,4 +255,18 @@ var _ = Describe("JUnit Reporter", func() {
 			})
 		})
 	}
+
+	When("output directory doesn't exist", func() {
+		It("should create before open file", func() {
+			output := "/tmp/not/exists/report.xml"
```

Comment:
Can we use `ioutil.TempDir()` here to create an empty directory and then reference a `non-existing-file.xml`. This allows the tests to be isolated, and also support systems that don't have a `/` directory structure.

Reply:
`/tmp` is another assumption about the correct place for temporary files, even on system that do support a `/` directory structure.

`ioutil.TempDir()` is more robust in this situation by checking for the value of `os.TempDir` - which is more likely to be correct on a given system.

Reply:
Thanks for your review. I fixed it by use `ioutil.TempDir` to create temp directory and use `filepath.Join` to join the outputfile.